hub-charts/edsc-hub/values.yaml

```diff
@@ -0,0 +1,78 @@
+jupyterhub:
+  hub:
+    nodeSelector:
+      cloud.google.com/gke-nodepool: core-pool
+    baseUrl: /edsc-hub/
+    annotations:
+      prometheus.io/scrape: "true"
+      # this needs to start with the value of `hub.baseUrl`
+      prometheus.io/path: /edsc-hub/hub/metrics
+    resources:
+      requests:
+        cpu: 200m
+        memory: 256Mi
+      limits:
+        cpu: 200m
+        memory: 256Mi
+  singleuser:
+    image:
+      # tag will be set by travis on deployment
+      name: earthlabhubops/ea-k8s-user-edsc-hub
+      tag: set-on-deployment
+    startTimeout: 600
+    cpu:
+      guarantee: 0.1
+      limit: 1.
+    memory:
+      guarantee: 500M
+      limit: 500M
+    lifecycleHooks:
+      postStart:
+        exec:
+          command:
+            - "sh"
+            - "-c"
+            - >
+              gitpuller https://github.com/earthlab-education/spring-2020-course-notebooks master ea-2020-class-notebooks;
```

Comment:
```suggestion
              gitpuller https://github.com/earthlab/edsc-summer-2020 master edsc-resources;
```

Reply:
@kcranston thank you for this pr. i'm just updating with the proper edsc repo for notebooks and such! @nkorinek this is how we pull in the notebooks in the repo that you created!